code/script.js

```diff
@@ -26,22 +29,156 @@ const showMessage = (message, sender) => {
       </section>
     `
   }
+
   // This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box
   chat.scrollTop = chat.scrollHeight
 }
 
-// Starts here
+// Greeting from bot
 const greeting = () => {
-  showMessage(`Hello there, What's your name?`, 'bot')
-  // Just to check it out, change 'bot' to 'user' here 👆
+  showMessage(`Hello there, I'm your life coach and I'm here to help you become the best version of yourself. What's your name?`, 'bot')
 }
 
-// Set up your eventlisteners here
 
-// When website loaded, chatbot asks first question.
-// normally we would invoke a function like this:
-// greeting()
-// But if we want to add a little delay to it, we can wrap it in a setTimeout:
-// setTimeout(functionName, timeToWaitInMilliSeconds)
-// This means the greeting function will be called one second after the website is loaded.
-setTimeout(greeting, 1000)
+// Event listener and validation of name input
+
+sendButton.addEventListener('click', (e) => {
+  e.preventDefault()
+  const name = nameInput.value
+    if(name === "") {
+      showMessage('Please enter your name','bot')
+    }
+    else {
+      showMessage(`${name}`,'user')
+      showCoachAreas(name)
+  }}
+  )
+    
+//Function for displaying available times
+
+const appointment = () => {
+  inputWrapper.innerHTML = `
+          <select id='select'>
+            <option value='' selected disabled>Select time and day</option>
+            <option value='Wednesday 4pm'>Wednesday 4pm</option>
+            <option value='Thursday 3pm'>Thursday 3pm</option>
+            <option value=-Friday 1pm'>Friday 1pm</option>
+          </select>
+        `
+        return inputWrapper
+  }
+  
+
+  //Function that displays new input bar after user chooses a time
+
+  const textInput = () => {
+    inputWrapper.innerHTML = `
+    <form id='name-form'>
+    <input id='name-input' type='text'/>
+    <button class='send-btn' type='submit'>
+            Send
+    </button>
+    </form>
+    `;
+
+  document.querySelector('.send-btn').addEventListener('click', (e) => {
+    e.preventDefault()
+    
+    let address = document.querySelector('#name-input').value
+    showMessage(address, 'user')
+    document.querySelector('#name-input').value = ''
+
+
+    setTimeout(()=> {
+      showMessage('Thanks, See you next week!', 'bot')
+    }, 2000)
+    
+  })
+      
```

Comment:
somewhere here you should disable your submit button because you can still send empty messages and your bot will repeat `Thanks, See you next week!`